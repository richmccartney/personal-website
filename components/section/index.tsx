import styled from '@emotion/styled'

const Section = styled.section`
  display: grid;
  grid-template-columns: repeat(12, [col] 1fr);
  gap: var(--px-size-24);
  grid-column: span 12;
  width: 100%;
  padding: var(--px-size-64) 0;

  @media (min-width: 768px) {
    padding: var(--px-size-128) 0;
  }
`

export default Section
