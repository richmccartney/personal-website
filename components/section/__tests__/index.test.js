import React from 'react'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'
import { matchers } from 'jest-emotion'

import section from '../index'

expect.extend(matchers)

describe('section:', () => {
  const wrapper = mount(<section>Hello world!</section>)

  test('renders default snapshot as expected', () => {
    const component = renderer.create(<section />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders as <section> as expected', () => {
    const container = wrapper.find('section')
    expect(container.is('section')).toBe(true)
  })
})
