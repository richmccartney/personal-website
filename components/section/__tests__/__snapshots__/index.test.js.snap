// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`section: renders default snapshot as expected 1`] = `<section />`;
